contracts/acl/IACLOracle.sol

```diff
@@ -6,5 +6,5 @@ pragma solidity ^0.4.24;
 
 
 interface IACLOracle {
```

Comment:
Let's include an `IACLOracleV1.sol` with the old interface.

Some existing apps, like the `TokenBalanceOracle`, will likely want to support both signatures for the time being.